docs/deployment/sso.md

```diff
@@ -21,10 +22,6 @@ This article walks you through single-sign on (SSO) configuration. Semgrep suppo
 
 ## OpenID Connect / OAuth 2.0
 
-:::warning
-Semgrep AppSec Platform does not support using OpenID with Microsoft Entra ID. Follow the instructions for [setting up SAML SSO with Microsoft Entra ID](#set-up-saml-sso-with-microsoft-entra-id) instead.
-:::
```

Comment:
I think we should retain this callout, and just link to the KB. We used to have a lot of people trying to do this, and the callout seems to have helped.

Reply:
Sounds good -- will bring this back